pom.xml

```diff
@@ -1048,6 +1048,12 @@
                 <artifactId>javassist</artifactId>
                 <version>3.22.0-GA</version>
             </dependency>
+
+            <dependency>
```

Comment:
This shouldn't be needed. presto-main already has a dependency on guava

Reply:
We don't specify the version of guava even in `presto-main`. Where should I specify the version 21.0 explicitly? `hmacMd5` etc was added since guava 20.0.

Reply:
It’s specified in the parent pom and is versio. 21.0